deps/build.jl

```diff
@@ -149,10 +150,41 @@ pm_cxx = chomp(read(`$perl $pm_config --cc`, String))
 jlcxx_cmake_dir = joinpath(CxxWrap.prefix_path(), "lib", "cmake", "JlCxx")
 julia_exec = joinpath(Sys.BINDIR , "julia")
 
+xcode_typeinfo_bug = false
+
+if (Sys.isapple())
+   # Lets have some fun for xcode 11.4 until
```

Comment:
I'd suggest that we instead make it explicit that we are working around a bug here that causes crashes
```suggestion
   # Work around a bug in Xcode 11.4 that causes segfaults, at least until
```

Reply:
applied but with SIGABRT instead of segfault